var/spack/repos/builtin/packages/amr-wind/package.py

```diff
@@ -29,9 +29,21 @@ class AmrWind(CMakePackage, CudaPackage, ROCmPackage):
     variant("shared", default=True, description="Build shared libraries")
     variant("tests", default=True, description="Activate regression tests")
     variant("tiny_profile", default=False, description="Activate tiny profile")
+    variant("hdf5", default=False, description="Enable HDF5 plots with ZFP compression")
+    variant("umpire", default=False, description="Enable Umpire")
+    variant("sycl", default=False, description="Enable SYCL backend")
+    variant("gpu-aware-mpi", default=False, description="gpu-aware-mpi")
 
     depends_on("hypre~int64@2.20.0:", when="+hypre")
     depends_on("hypre+mpi", when="+hypre+mpi")
+    depends_on("hdf5~mpi", when="+hdf5~mpi")
+    depends_on("hdf5+mpi", when="+hdf5+mpi")
+    depends_on("h5z-zfp", when="+hdf5")
+    depends_on("zfp", when="+hdf5")
```

Comment:
(Comment, not request) These can be grouped under `with when("+hdf5"):`  if it makes sense

Reply:
Okay if we add any more then we will update this.